docs/LanguageGuide.md

```diff
@@ -69,6 +69,36 @@ foo ; symbol
 (defmodule <name> <definition1> <definition2> ...) ;; The main way to organize your program into smaller parts
 ```
 
+### Conditional statements with 'cond'
+The cond statement executes a block of code if a specified condition is true. If the condition is false, another block of code can be executed. Examples are given below:
```

Comment:
I’d put `cond` in backticks here, to make sure it is rendered as a code element. What do you think?

Reply:
I'll do that and add the working of cond as well.Thank you so much.